src/sample-docs-per-year.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+"""
+take a sample of N documents per year.
```

Comment:
I think this docstring should be more detailed, we have so many sampling scripts

Reply:
This is a stratified sample, right? And can be used for records or motions.

Reply:
Some of this information can be added in the argparse help attributes though.